clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/Main.groovy

```diff
@@ -54,8 +53,10 @@ import java.security.Security
   GroovyTemplateAutoConfiguration,
   GsonAutoConfiguration,
   DataSourceAutoConfiguration,
-  ElasticsearchDataAutoConfiguration,
-  ElasticSearchRestHealthContributorAutoConfiguration
+  ElasticSearchJestHealthContributorAutoConfiguration,
```

Comment:
Curious why there are any elasticsearch-related changes here...but this line in particular appears to be causing a compile error [here](https://github.com/spinnaker/clouddriver/runs/4514289644?check_suite_focus=true):
```
> Task :clouddriver-web:compileGroovy FAILED
/home/runner/work/clouddriver/clouddriver/clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/Main.groovy: 56: Only classes and closures can be used for attribute 'exclude' in @org.springframework.boot.autoconfigure.EnableAutoConfiguration
 @ line 56, column 3.
     ElasticSearchJestHealthContributorAutoConfiguration,
     ^
```
In case it's relevant, https://github.com/spinnaker/clouddriver/pull/5586 went in recently with some nearby changes.

Reply:
This was some sort of change to make things compilable back when I first updated this. That class might also need to be moved to a string? Or a rebase here might be relevant, yeah.